doc-content/enterprise-only/getting-started/assets-types-ref.adoc

```diff
@@ -19,13 +19,13 @@ Business Processes are diagrams that describe the steps necessary to achieve bus
 Case management is an extension of Business Process Management (BPM) that enables you to manage adaptable business processes. Case management provides problem resolution for non-repeatable, unpredictable processes as opposed to the efficiency-oriented approach of BPM for routine, predictable tasks. It manages one-off situations when the process cannot be predicted in advance.
 +
 IMPORTANT: The business process application example includes features that are Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and are not recommended for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-For more information about Red Hat Technology Preview support, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+//For more information about Red Hat Technology Preview support, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 
-* Case definition (legacy)
+* Case Definition
```

Comment:
Case definition

I don't see any reason to cap. Goes for the who list. Data object. Decision table. Global variables. No (). The singular is implied in the plural.

Reply:
This is the UI usage. I will request a change.